MigrationCodelab/build.gradle

```diff
@@ -68,11 +68,25 @@ plugins {
 
 allprojects {
     repositories {
+        maven { // Compose alpha10 snapshot
+            url "https://androidx.dev/snapshots/builds/7067732/artifacts/repository/"
+        }
         google()
         jcenter()
     }
 }
 
+subprojects {
+    configurations.configureEach {
+        resolutionStrategy.eachDependency { details ->
+            if (details.requested.group.startsWith('androidx.compose')
+                    || details.requested.group.startsWith('androidx.ui')) {
```

Comment:
I think this can be simplified now that `androidx.ui` is renamed to `androidx.compose.ui`

Reply:
Sure! Will do that next time as this is throw-away code that will get removed in the next PR